server/server.js

```diff
@@ -11,11 +11,33 @@ const jsonwebtoken = require('jsonwebtoken');
 const cookieParser = require('cookie-parser')
 const ethereumutil = require("ethereumjs-util");
 const fs = require("fs");
+const Sentry = require('@sentry/node');
+const Tracing = require("@sentry/tracing");
 const PORT = process.env.PORT || 5000;
 
 require('dotenv').config({path : PATH.resolve(process.cwd(), '.env')});
 
 const APP = EXPRESS();
+
+Sentry.init({
+    dsn: "https://bd347d6c25b14066bfe38628a18ca151@o899113.ingest.sentry.io/5880381",
```

Comment:
let's put this into an environment variable

Reply:
done